source/guide/3.2/dsl-spec-relationships.md

```diff
@@ -190,6 +190,59 @@ The default configuration for `connection_type` is `all_to_all`.
 The same `connection_type` configuration can be applied to a `cloudify.relationships.contained_in` relationship type, though it will virtually have no effect.
 
 
+## *connection_type*: *all_to_all* and *all_to_one*
+As mentioned previously, the relationship types `cloudify.relationships.connected_to` and `cloudify.relationships.depends_on` and those that derive from it have a property named `connection_type` whose value can be either `all_to_all` or `all_to_one` (The default value is `all_to_all`).
+The following diagrams aim to make their semantics clearer.
```

Comment:
I think a use case should be added to each.
It's very clear why someone will want to connect all web apps to all databases.. while it's not too clear why they would want to connect to one. While it's clear to us that there are active-passive configurations (for instance), it might not be obvious to all.

I would suggest 2xOne-Liners explaining why you would use both.